### ISSUE TITLE
confusion about the code

### ISSUE THREAD

**Issue by chaopengz** [liuzechun/MetaPruning#20]
Hello zechun, First of all, appreciate  your  excellent work very much. It is very novelty.
Here are some questions about the code.

1. Why the code  of evaluation is almost same with training? Especially for the validation set, I see the path of validation set is absolutely same in the code of evaluation and training.  I think the path of validation set in the code of training is the sub-validation set apart from training and the path of validation set  in the code of evaluation is the real validation set of ImageNet. Is it right?

2. Is the MobileNetV1 [here](https://github.com/liuzechun/MetaPruning/blob/master/mobilenetv2/training/mobilenet_v2.py#L241) a typo?

**Comment by xuemo188** [comment]
大佬我想问一下这个数据集是imagenet具体版本

**Comment by liuzechun** [comment]
> Hello zechun, First of all, appreciate your excellent work very much. It is very novelty.
> Here are some questions about the code.
> 
>     1. Why the code  of evaluation is almost same with training? Especially for the validation set, I see the path of validation set is absolutely same in the code of evaluation and training.  I think the path of validation set in the code of training is the sub-validation set apart from training and the path of validation set  in the code of evaluation is the real validation set of ImageNet. Is it right?
> 
>     2. Is the MobileNetV1 [here](https://github.com/liuzechun/MetaPruning/blob/master/mobilenetv2/training/mobilenet_v2.py#L241) a typo?

Hi, chaopeng, thanks for your interest in my work! Sorry for my late reply.
1.the data path of the imagenet is just an example, I modified the data path in the released code for easy understand. The data split in my actual experiments follows the data split I mentioned in the paper.
2. Yes, it is a typo. Thank you so much for telling me that! I already fixed it. Sorry for the confusions I made.
If you have further questions, feel free to ask! :) I will reply you more promptly!
Thanks,
Zechun

**Comment by liuzechun** [comment]
> 大佬我想问一下这个数据集是imagenet具体版本

The version of the imagenet dataset I used in my experiment is Imagenet 2012.
